GoMeet/src/main/webapp/test/spec/spec-interactive-map.js

```diff
@@ -144,3 +144,38 @@ describe ('Build Info Window Vote', function() {
     expect(noteText).toBe(NOTE_A);
   });
 });
+
+/** Tests for displaying popular locations. */
+describe ('Display Popular Location', function() {
```

Comment:
Does the popular location list order the locations by number of votes? i.e. most popular location is number 1 on the list - or is it just top 3 in any order? Could be worth adding another test cases with vote count 5, 4, 3 and check the order of the locations in the list is as expected. Also, what happens if one of the locations has 0 votes but there are only 3 locations in total? Should/will it be included in the popular list?

Reply:
Covered in previous comment but only the locations with the high votes are listed (in no particular order).